src/components/Navigation/SearchNavigator.js

```diff
@@ -0,0 +1,36 @@
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import React from 'react';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import styles, {ICON_SIZE, INACTIVE_COLOR, PRIMARY_COLOR} from './styles';
+import Search from '../../screens/WIP/Search';
+import {SongInfo} from '../../screens/WIP/SongInfo';
+
+const Stack = createNativeStackNavigator();
+
+export function SearchNavigator() {
+  const SearchIcon = ({focused}) => (
+    <Icon
+      name="magnify"
+      size={ICON_SIZE}
+      color={focused ? PRIMARY_COLOR : INACTIVE_COLOR}
+    />
+  );
+
+  return (
+    <Stack.Navigator screenOptions={{headerShown: false}}>
+      <Stack.Screen
+        name={'Search'}
```

Comment:
You don't need to write curly brackets `{ }` when your prop value is a string.
```suggestion
        name='Search'
```